server/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:14
+
+WORKDIR /usr/src/app
+
+COPY package*.json ./
+COPY index.js ./
+
+RUN npm ci
```

Comment:
This could be altered slightly to take advantage of layer caching. Especially for dependencies:

```
COPY package.json package-lock.json /usr/src/app
RUN npm ci

COPY index.js
CMD ["node", "index.js"]
```

This would only redownload dependencies if a dependency changed in package.json. Otherwise - changes to index.js just get copied in without a full download
```